lib/disability_compensation/factories/api_provider_factory.rb

```diff
@@ -50,8 +50,8 @@ class UndefinedFactoryTypeError < StandardError; end
     }
   )
 
-  def self.call(**args)
-    new(**args).call
+  def self.call(**)
```

Comment:
Rubocop requires this change: `Use anonymous keyword arguments forwarding (`**`).`